home/templates/home/prerelease/session_list.html

```diff
@@ -25,7 +25,17 @@ <h1 class="text-5xl">Sessions</h1>
         </div>
         <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 pt-4">
             {% for session in sessions %}
+                {% if session.is_accepting_applications %}
```

Comment:
I'm not sure we want to hide past sessions. Though I do think we want these in the opposite order so that the first elements are the most recent.

Reply:
Is the placeholder will be displayed at the beginning of the other sessions?

Reply:
Yes I think that makes the most sense.